scout/run.py

```diff
@@ -3771,8 +4002,12 @@ def finalize_outputs(
         else:
             focus_yrs = self.handyvars.aeo_years
         # Initialize markets and savings totals across all ECMs
+
+        # Set total number of market variables that could be reported across ECMs
+        n_vars_all = 15
```

Comment:
Can this be retrieved from somewhere else and not hard-coded? ie, does it correspond to an existing dictionary or something?

Reply:
The value is equal to the number of variables that appear in the summary_vals list and aren't `metrics_finance` variables. It would not be especially straightforward to calculate because it is a subset of all the data available from `mkts` and `save` (plus `eff_capt`). If there were a simple way to count the number of key series combinations in `mkts` that start with "energy," "cost," or "carbon" and end in "baseline" or "efficient" plus the number of key series combinations in `save` that start with "energy" or "carbon," you'd get the right value, but that sounds like quite a few lines of code and embeds assumptions about the groups of data we are going to want to report out from `mkts` and `save`.

Reply:
Yes, this is what I ran into as well when coding that part. Perhaps with a more substantial refactor we could split out the summary list so that the financial metrics are separated out already?